deploy/docker-compose/docker-compose-latest.yml

```diff
@@ -0,0 +1,137 @@
+# Docker compose file which uses build context plus mounted volumes for easier local development
+# Note: storefront-fixed cannot use the build context because the .patch files are located in a parent directory of the Dockerfile
+version: '3'
+services:
+  agent:
+    image: "datadog/agent:7.29.0"
```

Comment:
Can we use a newer Agent image? We're using 7.31.1 in our newest labs.

Reply:
Yea for sure, let me update that